myScript.js

```diff
@@ -11,7 +11,6 @@ closeMobileItems.forEach(((element) => element.addEventListener('click', closeMo
 /* Project Detail objects */
 
 /* Contact Form validation */
-// eslint-disable-next-line no-useless-escape
 const regex = '[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$';
 document.querySelector('#form').addEventListener('submit', (e) => {
   for (let i = 0; i < e.target.length; i += 1) {
```

Comment:
- [ ] The error message should show up close to the "Get in touch" button. Please implement it